resolwe/flow/management/commands/register.py

```diff
@@ -342,8 +373,10 @@ def handle(self, *args, **options):
                 self.find_schemas(desc_path, schema_type=SCHEMA_TYPE_DESCRIPTOR, verbosity=verbosity))
 
         user_admin = users.first()
-        self.register_processes(process_schemas, user_admin, force, verbosity=verbosity)
         self.register_descriptors(descriptor_schemas, user_admin, force, verbosity=verbosity)
+        # NOTE: Descriptor schemas must be registered first, so
+        #       processes can validate 'entity_descriptor_schema' field.
+        self.register_processes(process_schemas, user_admin, force, verbosity=verbosity)
```

Comment:
Shouldn't we use logger instead of the verbosity and writing to stderr directly? I know this would require quite some refactoring...